securedrop_client/conversation/transcript/transcript.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Optional
+
+from securedrop_client import db as database
+
+from .items import Item
+from .items import transcribe as transcribe_item
+
+
+def transcribe(record: database.Base) -> Optional[Item]:
+    return transcribe_item(record)
+
+
+_ENTRY_SEPARATOR = "------\n"
+
+
+class Transcript:
+    def __init__(self, conversation: database.Source) -> None:
+
+        self._items = [transcribe(record) for record in conversation.collection]
```

Comment:
Note that this says `conversation.collection`, not `conversation.server_collection` :sunglasses:

Reply:
Filtering the conversation events that are relevant to the transcript is only the first step, of course, the second is sorting them. For now, sorting is provided by `database.Source`. (And I need to think more about what we'd need for `conversation.Transcript` to be in position to take responsibility for sorting its items.)